Makefile

```diff
@@ -33,14 +37,27 @@ clean:
 	-@ while read -r LINE; do [[ $${LINE} =~ test/.+ ]] && sudo rm -rf $${LINE}; done < .gitignore
 
 # -----------------------------------------------
-# --- Tests  & Lints ----------------------------
+# --- Tests  ------------------------------------
 # -----------------------------------------------
 
-tests:
-	@ ./test/bats/bin/bats --timing test/*.bats
+tests: ALWAYS_RUN
+	@ $(MAKE) generate-accounts tests/serial
+	@ $(MAKE) generate-accounts tests/parallel/set1
+	@ $(MAKE) generate-accounts tests/parallel/set2
+	@ $(MAKE) generate-accounts tests/parallel/set3
```

Comment:
I switched to this syntax as I encountered issues before when `generate-accounts` was not run before certain parallel tests were run because of prior modifications. Just FYI.

Reply:
BTW, do I miss something, where is the ALWAYS_RUN target? I only see references to it.

Reply:
At the top of the file; it is **not** defined as a target like `ALWAYS_RUN:`, but just as `.PHONY: ALWAYS_RUN` :)